tests/unit/test_biodata.py

```diff
@@ -0,0 +1,100 @@
+"""
+Tests the biodata module
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import unittest
+import json
+import os
+
+import ga4gh.datamodel as datamodel
+import ga4gh.datamodel.datasets as datasets
+import ga4gh.exceptions as exceptions
+import ga4gh.datamodel.biodata as biodata
+import ga4gh.protocol as protocol
+import ga4gh.avrotools as avrotools
+
+import tests.paths as paths
+
+
+class TestIndividuals(unittest.TestCase):
+    """
+    Tests the JSON Individuals class
+    """
+    dataset = datasets.AbstractDataset('dataset1')
+
+    def testToProtocolElement(self):
+        for localId in os.listdir(paths.testIndividualsDataDir):
```

Comment:
I started down that path, but decided against it when I observed `__init__` for datadriven since it intentionally skips `.json` elsewhere. https://github.com/ga4gh/server/blob/master/tests/datadriven/__init__.py#L45